### DISCUSSION TITLE
"Verified" badge is not visible on your organization's profile page

### DISCUSSION THREAD

**Discussion by fabiohcnobre** [community/community#54541] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

the "Verified" badge is not visible on your organization's profile page

![image](https://user-images.githubusercontent.com/22354660/236458653-127a8d37-77fa-4763-9e3c-2d363a833661.png)

**Comment by fbrettnich** [comment, marked as answer]
The prerequisite for the verified badge is that **all domains** on the profile are verified. The email address is using the domain "hotmail.com", which is not verified, and as a result, the verified badge does not appear. You should use an email address with your own domain in order to confirm it and receive the verified badge. Alternatively, you can remove the hotmail email address, and the verified badge should appear.

**Reply by fabiohcnobre** [reply]
Thanks !!!

![image](https://user-images.githubusercontent.com/22354660/236682991-ab203f51-55a5-49dc-82e4-903fafe3d0af.png)

**Comment by pcspl806** [comment]
> ### Select Topic Area
> Bug
> 
> ### Body
> the "Verified" badge is not visible on your organization's profile page
> 
> ![image](https://user-images.githubusercontent.com/22354660/236458653-127a8d37-77fa-4763-9e3c-2d363a833661.png)

I have also just verified my domain but the badge isn't showing up.
